vaadin-testbench-unit-junit5/src/test/java/com/vaadin/flow/component/virtuallist/VirtualListTesterTest.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (C) 2000-2024 Vaadin Ltd
+ *
+ * This program is available under Vaadin Commercial License and Service Terms.
+ *
+ * See <https://vaadin.com/commercial-license-and-service-terms> for the full
+ * license.
+ */
+package com.vaadin.flow.component.virtuallist;
+
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.router.RouteConfiguration;
+import com.vaadin.testbench.unit.UIUnitTest;
+import com.vaadin.testbench.unit.ViewPackages;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+@ViewPackages
+class VirtualListTesterTest extends UIUnitTest {
+
+    private VirtualListViewTester $virtualListView;
+
+    @BeforeEach
+    void init() {
+        RouteConfiguration.forApplicationScope()
+                .setAnnotatedRoute(VirtualListView.class);
+
+        $virtualListView = test(VirtualListViewTester.class, navigate(VirtualListView.class));
+    }
+
+    @Test
+    void virtualList_verifyComponents() {
+        Assertions.assertNotNull($virtualListView,
+                "Tester for VirtualListView not initialized.");
+
+        var $valueProviderVirtualList = $virtualListView.$valueProviderVirtualList();
+        Assertions.assertNotNull($valueProviderVirtualList,
+                "Tester for value provider VirtualList not initialized.");
+
+        var $componentRendererVirtualList = $virtualListView.$componentRendererVirtualList();
+        Assertions.assertNotNull($componentRendererVirtualList,
+                "Tester for component renderer VirtualList not initialized.");
+
+        var $callbackLitRendererVirtualList = $virtualListView.$callbackLitRendererVirtualList();
+        Assertions.assertNotNull($callbackLitRendererVirtualList,
+                "Tester for callback lit renderer VirtualList not initialized.");
+    }
+
+    @Test
+    void virtualList_verifyUserData() {
+        var users = UserData.all();
+
+        Assertions.assertEquals(UserData.USER_COUNT, users.size());
+    }
```

Comment:
Not sure about this test :thinking: 
I would probably remove it since it is not strictly related to the tester component.
Could it be an assertion (e.g. `assert users.size() == ...` in the `UserData.all()` method instead, if we really need it?